.github/workflows/docs.yml

```diff
@@ -21,11 +21,21 @@ on:
 jobs:
   sphinx:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9, "3.10"]
+
     steps:
     - uses: actions/checkout@v1
-    - uses: josh146/sphinx-action@master
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
       with:
-        docs-folder: "doc/"
-        repo-token: "${{ secrets.GITHUB_TOKEN }}"
-        pre-build-command: "apt install -y graphviz && pip3 install -r doc/requirements.txt && pip3 install . pip3 install openfermionpyscf"
-        build-command: "sphinx-build -b html . _build -W --keep-going"
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies
+      run: |
+        pip3 install -r doc/requirements.txt
+        pip3 install .
+        sudo apt install -y graphviz
+        make docs
```

Comment:
Would this change permanently?

Reply:
Nope! This was just to test the docs using different python versions. These changes should be removed before merging.